docs/backfill-toofull.md

```diff
@@ -0,0 +1,66 @@
+
+# Upmap fix for PGs with backfill toofull warnings
+
+```
+health: HEALTH_WARN
+10 nearfull osd(s)
+10 backfilltoofull osd(s)
+Low space hindering backfill (add storage if this doesn't resolve itself): 1 pg backfill_toofull
+```
+
+Problem:
+Usually when a node goes down or when draining capacity, there are some OSDs that become nearfull and eventually can lead to PGs being backfill_toofull warning pops up.
```

Comment:
`Usually when a node goes down or when draining capacity` seems broken. Should it be `Usually when a node goes down or when draining with limited capacity`?